proposal/comp110_architecture.bib

```diff
@@ -7,3 +7,91 @@ @article{shannon
 	number={3}, 
 	pages={379-423}
 }
+
+@ARTICLE{smith:launchpad, 
```

Comment:
This article looks like primary research describing an approach to level generation according to rhythm. :+1: